spec/observables/merge-spec.ts

```diff
@@ -31,237 +38,295 @@ describe('static merge(...observables)', () => {
 
-    const result = merge(e1);
+      const result = merge(e1);
 
-    expect(result).to.be.instanceof(Observable);
-    expectObservable(result).toBe('(abc|)');
+      expect(result).to.be.instanceof(Observable);
+      expectObservable(result).toBe('(abc|)');
+    });
   });
 
   it('should merge two lowerCaseO into RxJS Observable', () => {
-    const e1 = lowerCaseO('a', 'b', 'c');
-    const e2 = lowerCaseO('d', 'e', 'f');
+    rxTestScheduler.run(({ expectObservable }) => {
+      const e1 = lowerCaseO('a', 'b', 'c');
+      const e2 = lowerCaseO('d', 'e', 'f');
 
-    const result = merge(e1, e2);
+      const result = merge(e1, e2);
 
-    expect(result).to.be.instanceof(Observable);
-    expectObservable(result).toBe('(abcdef|)');
+      expect(result).to.be.instanceof(Observable);
+      expectObservable(result).toBe('(abcdef|)');
+    });
   });
 });
 
 describe('merge(...observables, Scheduler)', () => {
+  let rxTestScheduler: TestScheduler;
+
+  beforeEach(() => {
+    rxTestScheduler = new TestScheduler(observableMatcher);
+  });
+
   it('should merge single lowerCaseO into RxJS Observable', () => {
-    const e1 = lowerCaseO('a', 'b', 'c');
+    rxTestScheduler.run(({ expectObservable }) => {
+      const e1 = lowerCaseO('a', 'b', 'c');
 
-    const result = merge(e1, rxTestScheduler);
+      const result = merge(e1, rxTestScheduler);
 
-    expect(result).to.be.instanceof(Observable);
-    expectObservable(result).toBe('(abc|)');
+      expect(result).to.be.instanceof(Observable);
+      expectObservable(result).toBe('(abc|)');
+    });
   });
 });
 
 describe('merge(...observables, Scheduler, number)', () => {
+  let rxTestScheduler: TestScheduler;
+
+  beforeEach(() => {
+    rxTestScheduler = new TestScheduler(observableMatcher);
+  });
+
   it('should handle concurrency limits', () => {
-    const e1 =  cold('---a---b---c---|');
-    const e2 =  cold('-d---e---f--|');
-    const e3 =  cold(            '---x---y---z---|');
-    const expected = '-d-a-e-b-f-c---x---y---z---|';
-    expectObservable(merge(e1, e2, e3, 2)).toBe(expected);
+    rxTestScheduler.run(({ cold, expectObservable }) => {
+      const e1 = cold(' ---a---b---c---|            ');
+      const e2 = cold(' -d---e---f--|               ');
+      const e3 = cold('             ---x---y---z---|');
+      const expected = '-d-a-e-b-f-c---x---y---z---|';
+      expectObservable(merge(e1, e2, e3, 2)).toBe(expected);
+    });
   });
 
   it('should handle scheduler', () => {
-    const e1 =  of('a');
-    const e2 =  of('b').pipe(delay(20, rxTestScheduler));
-    const expected = 'a-(b|)';
+    rxTestScheduler.run(({ expectObservable, time }) => {
+      const delayTime = time('--|');
+      const e1 = of('a');
+      const e2 = of('b').pipe(delay(delayTime));
+      const expected = 'a-(b|)';
 
-    expectObservable(merge(e1, e2, rxTestScheduler)).toBe(expected);
+      expectObservable(merge(e1, e2, rxTestScheduler)).toBe(expected);
```

Comment:
Passing `rxTestScheduler` can be skipped.

Reply:
Oh, good catch, I didn't notice the test name. I'm so sorry for suggestion, this test should stay as it is.